tensorflow/python/ops/losses/losses_impl.py

```diff
@@ -419,6 +419,9 @@ def huber_loss(labels, predictions, weights=1.0, delta=1.0, scope=None,
     raise ValueError("predictions must not be None.")
   with ops.name_scope(scope, "huber_loss",
                       (predictions, labels, weights)) as scope:
+    if delta == float('inf'):
```

Comment:
This check only works if the delta is a python scalar infinity, not if it's a TF inf float.

Reply:
updated per your comment.